db/migrate/20160803021240_add_devise_to_users.rb

```diff
@@ -38,7 +38,7 @@ def self.up
 
     add_index :users, :email,                unique: true
     add_index :users, :reset_password_token, unique: true
-    add_index :users, :encrypted_password, unique: true
+    # add_index :users, :encrypted_password, unique: true
```

Comment:
Typically migrations should not be changed after they are created, rather a new migration would be made to undo the changes made. However, this database is odd and mostly reads off the `schema.rb` anyway, so this isn't a big deal.